Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN yum install -y epel-release && \
 ADD supervisord.conf /etc/supervisord.conf
 ADD make-nagios-fhservices-cfg make-nagios-commands-cfg fhservices.cfg.j2 commands.cfg.j2 /opt/rhmap/
 ADD plugins/default/* plugins/core/* /opt/rhmap/nagios/plugins/
```

Comment:
I haven't normally paid attention to that, but we should be preferring to use `COPY` instead of `ADD`:
https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/add-or-copy